src/main/java/com/gluonhq/attach/AttachArtifactResolver.java

```diff
@@ -54,10 +54,12 @@
                 .map(d -> {
                     if (UTIL_ARTIFACT.equals(d.getArtifactId())) {
                         if (Constants.PROFILE_ANDROID.equals(target) ||
-                            Constants.PROFILE_IOS.equals(target)||
+                            Constants.PROFILE_IOS.equals(target) ||
                             Constants.PROFILE_IOS_SIM.equals(target)) {
+                            String utilTarget = Constants.PROFILE_IOS_SIM.equals(target) ?
+                                    Constants.PROFILE_IOS : target;
                             return new DefaultArtifact(d.getGroupId(), d.getArtifactId(),
-                                    target, d.getType(), d.getVersion());
+                                    utilTarget, d.getType(), d.getVersion());
```

Comment:
This means the same target is used, but the native code is built for a different architecture. Or do we assume lipo is used to create multi-arch libraries?

Reply:
We use `lipo` indeed.